src/modules/backupRestore/backupRestoreEntities/entities/languageVariants.ts

```diff
@@ -44,7 +44,23 @@ export const languageVariantsEntity = {
   serializeEntities: JSON.stringify,
   deserializeEntities: JSON.parse,
   importEntities: async (client, { entities: fileVariants, context, logOptions }) => {
-    await serially(fileVariants.map(createImportVariant(client, context, logOptions)));
+    await serially(
+      fileVariants
+        .toSorted((a, b) => {
+          if (a.item.id === b.item.id) {
+            if (a.language.id === emptyId) {
+              return -1;
+            }
+
+            if (b.language.id === emptyId) {
```

Comment:
I don't think this can ever happen. Both of these `if`s are wrapped in `aId === bId` and the first `if` returns when `aId` === `emptyId` and if that's false `bId` cannot be empty as well, because in this branch you can only be when `aId` === `bId`. My guess is that the top `if` should have `!==` instead.

Reply:
It's ease to make mistake in these `if`s. If we have `ts-pattern` installed here, I'd suggest using it here. Something like
```
match([a.item.id, b.item.id])
  .with([emptyId, emptyId], () => 0)
  .with([emptyId, P._], () => -1)
  .with(P._, emptyId], () => 1)
  .otherwise(() => 0)
```
could work.